composer.json

```diff
@@ -9,6 +9,8 @@
     "require": {
         "php": ">=5.5",
         "ext-curl": "*",
+        "guzzlehttp/guzzle": "~4.0",
```

Comment:
We can't upgrade as the Jira API still uses the old version.

/cc @AlexKovalevych